src/js/components/Calendar.js

```diff
@@ -129,9 +131,11 @@ export default class Calendar extends Component {
     } else {
       event.preventDefault();
       event.stopPropagation();
-      var previousDay = moment(this.state.current).subtract(1, 'days');
-      if (! previousDay.isSame(this.state.reference, 'month')) {
-        this.setState({reference: this.state.reference.subtract(1, 'month'), current: previousDay});
+      const { current, reference } = this.state;
```

Comment:
can we move `const { current, reference } = this.state;` to be outside the if/else scenario.

BTW, thanks for working on the ES6 refactoring.